environments/azure-dyn.yaml

```diff
@@ -190,60 +190,60 @@ of:
     enabled: true
     annotations: 
       kubernetes.io/ingress.class: azure/application-gateway
-      appgw.ingress.kubernetes.io/appgw-ssl-certificate: dev-buying-catalogue-digital-nhs-uk 
+      appgw.ingress.kubernetes.io/appgw-ssl-certificate: dyn-buying-catalogue-digital-nhs-uk 
       appgw.ingress.kubernetes.io/ssl-redirect: "true"
     hosts:
-      - host: <namespace>.dyn.buyingcatalogue.digital.nhs.uk #To override from build
+      - host: <namespace>.dynamic.buyingcatalogue.digital.nhs.uk
         paths: 
           - /order/*
     tls: []  
   baseUri: "/order"  
-​
+
 redis:
   enabled: false
-  disabledUrl:                                            # Override per environment
+  disabledUrl:
   cluster:
     enabled: false
   usePassword: false
-​
+
 redis-commander:
   enabled: false
   urlPrefix: redis-commander
   ingress:
     enabled: true
     annotations: 
       kubernetes.io/ingress.class: azure/application-gateway
-      appgw.ingress.kubernetes.io/appgw-ssl-certificate: dev-buying-catalogue-digital-nhs-uk 
+      appgw.ingress.kubernetes.io/appgw-ssl-certificate: dyn-buying-catalogue-digital-nhs-uk 
       appgw.ingress.kubernetes.io/ssl-redirect: "true"
     hosts:
-      - host: <namespace>.dyn.buyingcatalogue.digital.nhs.uk #To override from build
+      - host: <namespace>.dynamic.buyingcatalogue.digital.nhs.uk
         paths: 
           - /redis-commander/*
-​
+
 pb-ac-tests:
-  enabled: true
+  enabled: false
   testFilter: SmokeTest
-​
+
 mp-ac-tests:
-  enabled: true
+  enabled: false
   testFilter: SmokeTest
-​
+
 admin-ac-tests:
-  enabled: true
+  enabled: false
```

Comment:
What's the reasoning behind disabling all the AC tests?

Reply:
These were disabled to reduce load on low spec cluster. Updated now as next task will be to set these up